plugins/wpe-headless/includes/admin-menus/callbacks.php

```diff
@@ -24,6 +25,12 @@ function wpe_headless_remove_admin_menu_pages() {
 	 */
 	global $submenu;
 
+	$disable_theme = wpe_headless_get_setting( 'disable_theme', false );
```

Comment:
Curious, what's the rationale behind adding the `false` here? I see that it's the default value for the argument.

I see that the value is set to `1` on activation. If it's unchecked, does the value get removed from the settings array entirely?

Reply:
>I see that the value is set to 1 on activation. If it's unchecked, does the value get removed from the settings array entirely?

Yes, unchecked and the value will be removed from the settings, which is why the default is `false` and not `true`. Else, it would constantly return true.

Reply:
And `false` is the default value for the `$default` parameter in `wpe_headless_get_setting()`, I essentially could just remove the second parameter.

Reply:
Got it, makes sense! I like how it's explicit. I suppose my only feedback here would be to have a getter function for all of these toggles like you already do for `wpe_headless_domain_replacement_enabled()`.